### DISCUSSION TITLE
Simple in React, impossible in Svelte?

### DISCUSSION THREAD

**Discussion by lkj4** [sveltejs/kit#4848] [Q&A] [answered]
How can I turn `this _random_ string` into `<div>this <i>random</i> string</div>` (without knowing in advance which word, if at all, is written italic)?

**Comment by bluwy** [comment, marked as answer]
Use a markdown renderer to turn the string into html and use [`{@html}`](https://svelte.dev/tutorial/html-tags).

**Reply by lkj4** [reply]
Thanks for your answer, I've the the lib svelte-markdown but I just want to transform one and only tag, the entire lib is too heavy weight. 

How could I transform my aforementioned example? I know that in React it's just one line.

**Reply by tnzk** [reply]
I assume it would be easier for anyone to answer if you share the example in React.

**Reply by maku** [reply]
here is a simple example:  [https://svelte.dev/repl/b783c3f7500a43508410b7878e82e7f3?version=3.48.0 ]() -> of course when you don't want to use a lib (like "showdown" in this example) you have to write the "conversion" function by yourself (but that would be the same function like in react)

**Reply by tnzk** [reply]
I suppose [this](https://svelte.dev/repl/413546406e4d4ddfa77adae8691f0fe7?version=3.48.0) based on @maku example is as simple enough as in React.

**Reply by lkj4** [reply]
@tnzk @maku @bluwy Thanks for looking into this. And I was wrong with React. You could use `dangerouslySetInnerHTML` it's one regex but it wouldn't be safe, so it wouldn't be easier than with Svelte. I'll check out showdown and other converters! 🙂
